cwrap/prototype.py

```diff
@@ -54,8 +74,12 @@ def _registerType(type_name, type_class_or_function, is_return_type=True, storag
 _registerType("long", ctypes.c_long)
 _registerType("long*", ctypes.POINTER(ctypes.c_long))
 _registerType("char", ctypes.c_char)
-_registerType("char*", ctypes.c_char_p)
-_registerType("char**", ctypes.POINTER(ctypes.c_char_p))
+if six.PY2:
+    _registerType("char*", ctypes.c_char_p)
+    _registerType("char**", ctypes.POINTER(ctypes.c_char_p))
+if six.PY3:
+    _registerType("char*", UTF8String)
+    # _registerType("char**", ctypes.POINTER(UTF8String))
```

Comment:
Commented out code!

Reply:
This has changed a lot now, and will still. I'll take a look at these comments again when it's less WIP